packages/rules-engine/src/lib/rule-engine.spec.ts

```diff
@@ -20,10 +17,19 @@ test('Simple Server Workflow', (t) => {
   const rule1 = engine.logicRule(someRule);
   const rule2 = engine.logicRule({ '<': [{ var: 'answer1' }, 10000] });
   const data = { answer1: 70 , answer2: true};
-  
-  const isValid = rule1.evaluate(data) && rule2.evaluate(data) 
+
+  const isValid = rule1.evaluate(data) && rule2.evaluate(data)
```

Comment:
*non-blocker - should soon refactor to some iterative variant (`evaluate` n rules)